app/presenters/event_presenter.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Presenter for SDR events.
+class EventPresenter < SimpleDelegator
+  def label
+    I18n.t("events.#{event_type}", default: event_type.humanize)
+  end
+
+  def who
+    data['who']
+  end
```

Comment:
Where is `data` defined?

Reply:
In the event passed to SimpleDelegator.

Let me see if I can make that a tad clearer.